InputBarAccessoryView/Controls/InputBarButtonItem.swift

```diff
@@ -29,30 +29,30 @@ import UIKit
 
 /**
  A InputItem that inherits from UIButton
- 
+
```

Comment:
Looks like you have added some spaces here, could you please clean that up? I use Xcode's automatically trim trailing whitespace setting. (for lining purposes)

Reply:
This was probably introduced by a file wide Xcode `CTRL-i`, I also have that setting. I usually ignore whitespace on commits so it slipped through.